### PR TITLE
Update lifecycle page to most recent changes

### PR DESCRIPTION
**Pull request by Lawouach**
This works with recent changes of rug, the cli and the various editors.

### CODE REVIEW

File: docs/user-guide/rug/lifecycle.md [hunk 1]
```diff
@@ -55,41 +54,42 @@ The created Rug project has the following directory structure:
...
 $ cat manifest.yml
-group: atomist-rugs
-artifact: my-rug-project
+group: "atomist-rugs"
+artifact: "my-rug-project"
 version: "0.1.0"
-requires: "[0.8.0,1.0.0)"
-dependencies:
-extensions:
+requires: "[0.16.0,1.0.0)"
```
Comment (kipz) [inline docs/user-guide/rug/lifecycle.md]: Might be worth putting 0.17.3 in there for good measure?

File: docs/user-guide/rug/lifecycle.md [hunk 2]
```diff
@@ -55,41 +54,42 @@ The created Rug project has the following directory structure:
...
 ```
 
 The `manifest.yml` specifies the unique coordinates of the Rug archive
-as well as its version.  Dependencies and extensions can also be
-declared
+as well as its version.  Dependencies and extensions may also be
```
Comment (kipz) [inline docs/user-guide/rug/lifecycle.md]: The version is considered one of the coordinates, so we should reword this.

File: docs/user-guide/rug/lifecycle.md [hunk 3]
```diff
@@ -127,45 +132,36 @@ run from any directory, you must install it:
...
-Successfully installed archive for atomist-rugs:my-rug-project:0.1.0
 ```
 
 The `install` command takes the project, packages it up for
 distribution, and installs it under `~/.atomist/repository`.
 
-The contents of the archive can be further tuned by adding entries to
-`.gitignore` and `.atomist/ignore`, the packaging step considers the
-contents of these files when creating the archive.
+The contents of the archive can be further tuned by adding entries to
```
Comment (kipz) [inline docs/user-guide/rug/lifecycle.md]: extraneous space.

File: docs/user-guide/rug/lifecycle.md [hunk 4]
```diff
@@ -193,46 +189,32 @@ Once you have your publishing repository configured, you simply run
...
-  Downloading atomist-rugs/my-rug-project/maven-metadata.xml ← rugs-release (381 bytes) succeeded
-  Uploading atomist-rugs/my-rug-project/maven-metadata.xml → rugs-release (333 bytes) succeeded
+  Uploading atomist-rugs/my-rug-project/0.1.0/my-rug-project-0.1.0.zip → rugs-release (530 kb) succeeded                                                                                                  
+  Uploading atomist-rugs/my-rug-project/0.1.0/my-rug-project-0.1.0.pom → rugs-release (640 bytes) succeeded                                                                                               
+  Uploading atomist-rugs/my-rug-project/0.1.0/my-rug-project-0.1.0-metadata.json → rugs-release (700 bytes) succeeded                                                                                     
+  Downloading atomist-rugs/my-rug-project/maven-metadata.xml ← rugs-release (381 bytes) succeeded                                                                                                         
+  Uploading atomist-rugs/my-rug-project/maven-metadata.xml → rugs-release (333 bytes) succeeded
```
Comment (ddgenome) [inline docs/user-guide/rug/lifecycle.md]: I wonder why these lines have so many spaces at the end?

File: docs/user-guide/rug/lifecycle.md (hunk 2 above)
Comment (ddgenome) [inline docs/user-guide/rug/lifecycle.md]: Something like

```markdown
The `manifest.yml` specifies the unique coordinates of the Rug archive:
its name, group, and version.  Dependencies...
```

**Pull request merged**